### PR TITLE
Breaking/xref host

### PR DESCRIPTION
**Pull request by live1206**
https://dev.azure.com/ceapex/Engineering/_workitems/edit/134114

pair with 
https://ceapex.visualstudio.com/Engineering/_git/Docs.Build/pullrequest/17521?_a=overview

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5239)

### CODE REVIEW

File: src/docfx/build/xref/XrefResolver.cs [hunk 1]
```diff
@@ -32,6 +33,7 @@ internal class XrefResolver
 
             _dependencyMapBuilder = dependencyMapBuilder;
             _fileLinkMapBuilder = fileLinkMapBuilder;
+            _xrefHostName = context.Config.XrefHostName;
```
Comment (yufeih) [inline src/docfx/build/xref/XrefResolver.cs]: fallback to host name in `base_url` in case this config is not set?

**Comment by yufeih** [comment]
#4450

File: src/docfx/build/xref/XrefResolver.cs (hunk 1 above)
Comment (live1206) [reply src/docfx/build/xref/XrefResolver.cs]: sounds good

File: docs/specs/xref.yml [hunk 2]
```diff
@@ -106,6 +106,7 @@ repos:
     - files:
         docfx.yml: |
           baseUrl: https://docs.com/base_path
+          xrefHostName: https://test.docs.com
```
Comment (yufeih) [inline docs/specs/xref.yml]:
```suggestion
          xrefBaseUrl: https://test.docs.com
```

**Pull request merged**